### ISSUE TITLE
blockUI: false not working in 7.1

### ISSUE THREAD

**Issue by jaffadog** [FrDH/jQuery.mmenu#932]
When blockUI is set to false, then glbl.$blck does not get set, and this blows up:

````
this.bind( 'setPage:after',
	function( $page )
	{
		glbl.$blck
			.children( 'a' )
			.attr( 'href', '#' + $page.attr( 'id' ) );
	}
);
````

**Comment by FrDH** [comment]
Thank you for reporting this issue.
I'll fix it in the next release.

**Comment by FrDH** [comment]
Should be fixed in 7.2
